docs/ntd-api.md

```
@@ -428,10 +510,28 @@ PUT /api/backup/database/auto
 ```
 
+| 字段 | 类型 | 必填 | 说明 |
+|------|------|------|------|
+| `enabled` | boolean | 是 | 是否启用自动备份 |
+| `cron` | string | 是 | Cron 表达式（6 字段：秒 + 标准 5 字段） |
+| `max_files` | number | 否 | 保留的最大备份文件数（默认 30，0 非法） |
+
+---
+
+### 优化数据库
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)